projects/OmniSource/README.md

```diff
@@ -0,0 +1,33 @@
+# Omni-sourced Webly-supervised Learning for Video Recognition
```

Comment:
what's the purpose of the projects folder? does it accept public pull requests?
will it be an exhibition?

what do you think? cc. @hellock

Reply:
i'm thinking that, how about put the configs and readme under `configs/weaklysupervised/omnisource`?

although the method might have used slowonly as the backbone, but it does not really belongs to the work of slowfast.
what do you think, @kennymckormick

Reply:
Actually, in this PR, we only release checkpoints of OmniSource. The training configs are not included (since we haven't released data & code). I think it's OK to put those checkpoints in the TSN and SlowOnly directory, since they are the architectures of these checkpoints. If we put them in another directory: 1. We do not have training configs now. 2.  Few people will walk in that directory and see our work.

Reply:
okay, then how about put the readme in configs folder (and leave the inference configs where they are now)?

Reply:
All right, I plan to put the readme in the configs/OmniSource folder, add training configs to that folder after we support OmniSource training in MMAction2